examples/custom_extern/example.p4

```diff
@@ -1,3 +1,18 @@
+/* Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
you need to change this line to
```
/* Copyright 2022 <WHATEVER IS APPROPRIATE IN YOUR CASE>
```
or potentially:
```
/* Copyright 2022 P4lang Authors
```

same in other files

Reply:
Fixed.